cmd/create/cluster/cmd.go

```diff
@@ -492,6 +518,21 @@ func run(cmd *cobra.Command, _ []string) {
 		isIAM = !isSTS
 	}
 
+	mode := args.mode
+	if interactive.Enabled() && isSTS {
```

Comment:
do we want this to always be on, or do we want the mode to be only passed explicitly. Issue being manual is not the cleanest at the moment. 
We can run the manual create operator roles and get the aws outputs, but we can not do the same for the oidc providers as we need the operator roles to exists. Would it be better if manual mode is selected we output to run the rosa commands to create the operator roles and oidc providers. Where as if auto mode is selected it is safe to run both create operator roles and oidc providers.

Reply:
I think we want to be explicit. I would say don't prompt for `mode` in here and then just show which `rosa` commands the user should run after creating cluster. If they explicitly set `mode`, then we can show the `aws` commands instead.